consensus/validation.go

```diff
@@ -36,14 +36,15 @@ type SignedMessageValidator interface {
 }
 
 type defaultMessageValidator struct {
-	allowHighNonce bool
+	allowHighNonce          bool
+	bypassBLSSignatureCheck bool
 }
 
 // NewDefaultMessageValidator creates a new default validator.
 // A default validator checks for both permanent semantic problems (e.g. invalid signature)
 // as well as temporary conditions which may change (e.g. actor can't cover gas limit).
 func NewDefaultMessageValidator() SignedMessageValidator {
-	return &defaultMessageValidator{}
+	return &defaultMessageValidator{bypassBLSSignatureCheck: true}
```

Comment:
Please TODO and link to an issue for changing this.

Reply:
Where does this come from?  Shouldn't we be validating individual message signatures?